scripts/controls.js

```diff
@@ -22,17 +22,28 @@ export default class Controls {
     if (!tokenButton) return;
 
     tokenButton.tools.push(this._convenientEffectsAppButton);
-    tokenButton.tools.push(this._removeEffectsButton);
+    if (this._settings.unifiedAppButton === 'none') tokenButton.tools.push(this._removeEffectsButton);
   }
 
   get _convenientEffectsAppButton() {
+    const title = this._settings.unifiedAppButton !== 'none' && game.user.role >= this._settings.removeControlsPermission
+      ? `<center><b>DFreds CE Apps</b></center><hl>
```

Comment:
Probably a good idea to not use the CE abbreviation at all. Convenient Effects is a little verbose but it's at least clear

Reply:
Is this any better? 
![image](https://github.com/DFreds/dfreds-convenient-effects/assets/7237090/337f4690-f8f0-4f41-b647-75eed6d756c2)